compiler/rustc_middle/src/ty/instance.rs

```diff
@@ -93,6 +93,14 @@ pub enum InstanceDef<'tcx> {
     ///
     /// The `DefId` is for `Clone::clone`, the `Ty` is the type `T` with the builtin `Clone` impl.
     CloneShim(DefId, Ty<'tcx>),
+
+    /// Compiler-generated `<T as Default>::default` implementation.
+    ///
+    /// Function definitions and closures without upvars have a compiler-generated `Default`
+    /// implementation.
+    ///
+    /// The `DefId` is for `Default::default`, the `Ty` is the type `T` with the builtin `Default` impl.
```

Comment:
```suggestion
    /// The `DefId` is for `Default::default`, the `Ty` is the type `T` with the built-in `Default` impl.
```
there is inconsistency thoughout in the hyphenation of the(se) words in this PR (and others), the surface area is limited to documentation so it maybe worth a simple rg and replace for consistency?

Reply:
Yeah this is just from rebasing on top of other PRs that changed these things, will address that before this get merged.